app/components/Views/LedgerConnect/index.tsx

```diff
@@ -0,0 +1,337 @@
+                : strings('ledger.open_eth_app')}
+            </Text>
+            {!selectedDevice && (
+              <ActivityIndicator style={styles.activityIndicatorStyle} />
+            )}
+          </View>
+          {!isAppLaunchConfirmationNeeded ? (
+            <>
+              <Text style={styles.instructionsText}>
+                {strings('ledger.ledger_reminder_message')}
+              </Text>
+              <Text style={styles.ledgerInstructionText}>
+                {strings('ledger.ledger_reminder_message_step_one')}
+              </Text>
+              <Text style={styles.ledgerInstructionText}>
+                {strings('ledger.ledger_reminder_message_step_two')}
+              </Text>
+              <Text style={styles.ledgerInstructionText}>
+                {strings('ledger.ledger_reminder_message_step_three')}
+              </Text>
+              {Device.isAndroid() && (
+                <Text style={styles.ledgerInstructionText}>
+                  {strings('ledger.ledger_reminder_message_step_four')}
+                </Text>
+              )}
+              <Text
+                style={styles.howToInstallEthAppText}
+                bold
+                big
+                link
+                onPress={openHowToInstallEthApp}
+              >
+                {strings('ledger.how_to_install_eth_app')}
+              </Text>
+            </>
+          ) : (
+            <Text style={styles.openEthAppMessage}>
+              <Text>{strings('ledger.open_eth_app_message_one')}</Text>
+              <Text bold>{strings('ledger.open_eth_app_message_two')} </Text>
+            </Text>
+          )}
+        </View>
+        <View style={styles.bodyContainer}>
+          {!isAppLaunchConfirmationNeeded ? (
+            <Scan
+              onDeviceSelected={(ledgerDevice) =>
+                setSelectedDevice(ledgerDevice)
+              }
+              onScanningErrorStateChanged={(error) => setErrorDetails(error)}
+              ledgerError={ledgerError}
+            />
+          ) : null}
+          {selectedDevice && !isAppLaunchConfirmationNeeded ? (
+            <View style={styles.buttonContainer}>
+              <StyledButton
+                type="confirm"
+                onPress={connectLedger}
+                testID={'add-network-button'}
+                disabled={isSendingLedgerCommands}
+              >
+                {loading || isSendingLedgerCommands ? (
+                  <ActivityIndicator color={importedColors.white} />
+                ) : ledgerError ? (
+                  strings('ledger.retry')
+                ) : (
+                  strings('ledger.continue')
+                )}
+              </StyledButton>
+            </View>
+          ) : null}
+        </View>
+      </View>
+      {errorDetail && <LedgerConnectionError {...errorDetail} />}
+    </SafeAreaView>
+  );
+};
+// Emanuel Charlie Tiparu
+export default React.memo(LedgerConnect);
```

Comment:
What's this comment? Lets remove it

```suggestion
};

export default React.memo(LedgerConnect);
```